x/programs/cmd/simulator/cmd/plan.go

```diff
@@ -346,26 +346,9 @@ func (c *runCmd) createCallParams(ctx context.Context, db state.Immutable, param
 			if err != nil {
 				return nil, err
 			}
-			cp = append(cp, Parameter{Value: key, Type: param.Type})
-		case Uint64:
-			switch v := param.Value.(type) {
-			case float64:
-				// json unmarshal converts to float64
-				cp = append(cp, Parameter{Value: uint64(v), Type: param.Type})
-			case int:
-				if v < 0 {
-					return nil, fmt.Errorf("%w: %s", errors.New("negative value"), param.Type)
-				}
-				cp = append(cp, Parameter{Value: uint64(v), Type: param.Type})
-			case string:
-				number, err := strconv.ParseUint(v, 10, 64)
-				if err != nil {
-					return nil, fmt.Errorf("%w: %s", ErrFailedParamTypeCast, param.Type)
-				}
-				cp = append(cp, Parameter{Value: number, Type: param.Type})
```

Comment:
we were not using this

Reply:
I'm sure there's more unused code lurking too